cms/taxonomy/mixins.py

```diff
@@ -35,9 +35,10 @@ def clean(self) -> None:
         for exclusive_sub_page_type in ExclusiveTaxonomyMixin.__subclasses__():
             # Check if other pages are exclusively linked to this topic.
             # Translations of the same page are allowed, but other pages aren't.
-            # TODO for multilingual support, this will need to exclude different language versions of the same page by
-            # excluding matching translation_keys
-            qs = exclusive_sub_page_type.objects.filter(topic=self.topic_id)  # type: ignore[attr-defined]
+            qs = exclusive_sub_page_type.objects.filter(  # type: ignore[attr-defined]
+                topic=self.topic_id,
+                locale=self.locale,  # type: ignore[attr-defined]
```

Comment:
```suggestion
                locale=self.locale_id,  # type: ignore[attr-defined]
```

to keep with tradition

Reply:
Addressed in [99db1b3](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/99db1b3e24591fccea192c4a305f5199cb698de7)